src/azure-cli/azure/cli/command_modules/backup/_help.py

```diff
@@ -181,7 +197,36 @@
 short-summary: Create a new policy for the given BackupManagementType and workloadType.
 examples:
   - name: Create a new policy for the given BackupManagementType and workloadType.
-    text: az backup policy create --policy {policy} --resource-group MyResourceGroup --vault-name MyVault --name MyPolicy
+    text: az backup policy create --policy {policy} --resource-group MyResourceGroup --vault-name MyVault --name MyPolicy --backup-management-type AzureStorage
+"""
+
+helps['backup protectable-item'] = """
+type: group
+short-summary: An item which can be protected or backed up to an Azure Recovery services vault with an associated policy.
```

Comment:
Can you refine your short summary here?
e.g. Manage the item which can be protected or backed up to an Azure Recovery services vault with an associated policy.
or 
Manage the item in an Azure Recovery services vault with an associated policy.

Reply:
@Juliehzl :  This summary was for defining "What is a protectable item" and hence it says "An item which can be protected......" Please correct me if my assumption is wrong.

Reply:
Typically, the summary is used to describe what the command or command group do. IN this way, we may use something like "Mange xxx resource" as short summary for a command group.

Reply:
Ok. @sambitratha : Then let's have the summary at a group level as "Manage the item which is yet to be protected or backed up to an Azure Recovery services vault with an associated policy"